File: src/test/java/cernx/colt/matrix/tfloat/impl/SparseFloatMatrix2DViewTest.java
package cernx.colt.matrix.tfloat.impl;

public class SparseFloatMatrix2DViewTest extends SparseFloatMatrix2DTest {

    public SparseFloatMatrix2DViewTest(String arg0) {
        super(arg0);
    }

    protected void createMatrices() throws Exception {
        A = new SparseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        B = new SparseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose();
        Bt = new SparseFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose();
    }
}

File: src/test/scala/cern/colt/matrix/tfloat/impl/SparseFloatMatrix2DViewTest.scala
package cern.colt.matrix.tfloat.impl

import cern.colt.matrix.MatrixTypes.SparseFloatMatrix2D

class SparseFloatMatrix2DViewTest(arg0: String) extends SparseFloatMatrix2DTest(arg0) {

  override protected def createMatrices() {
    A = new SparseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    B = new SparseFloatMatrix2D(NCOLUMNS, NROWS).viewTranspose()
    Bt = new SparseFloatMatrix2D(NROWS, NCOLUMNS).viewTranspose()
  }
}
